choose-an-avatar.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Henny+Penny|Montserrat+Alternates">
+  <link rel="stylesheet" href="node_modules/materialize-css/bin/materialize.css" media="screen,projection"/>
+  <link rel="stylesheet" href="main.css">
+
+  <title>Welcome to CAMHS</title>
+</head>
+<body>
+  <div class="avatar">
+    <h1 class="avatar__title">Choose an animal</h1>
+    <div class="carousel">
+      <a class="carousel-item" href="#one!"><img src="assets/lion.svg"></a>
+      <a class="carousel-item" href="#two!"><img src="assets/lion.svg"></a>
+      <a class="carousel-item" href="#three!"><img src="assets/lion.svg"></a>
+    </div>
+  </div>
+
+  <script src="node_modules/jquery/dist/jquery.min.js"></script>
+  <script src="node_modules/materialize-css/bin/materialize.js"></script>
```

Comment:
It kinda depends how you want to deploy this. If you are putting it on heroku you can just make your server find these files (when heroku deploys it will npm i so the files will be available). If you are gonna put it on gh pages these files will need to be commited to gh-pages branch. Ideally these files would not be commited to master, so I like where you are going with this